controllers/state_manager.go

```diff
@@ -161,10 +161,11 @@ type ClusterPolicyController struct {
 	openshift        string
 	ocpDriverToolkit OpenShiftDriverToolkit
 
-	runtime        gpuv1.Runtime
-	hasGPUNodes    bool
-	hasNFDLabels   bool
-	sandboxEnabled bool
+	runtime            gpuv1.Runtime
+	runtimeSupportsCDI bool
```

Comment:
Is `gpuv1.Runtime` just a string. Does it make sense to update that type with `SupportsCDI` and `Version` menbers instead of having separate members here?